Security/PermissionCheckboxSetField.php

```diff
@@ -279,7 +279,7 @@ public function saveInto(DataObjectInterface $record) {
 		// Remove all "privileged" permissions if the currently logged-in user is not an admin
 		$privilegedPermissions = Permission::config()->privileged_permissions;
 		if(!Permission::check('ADMIN')) {
-			foreach($this->value as $id => $bool) {
+			if (is_array($this->value)) foreach($this->value as $id => $bool) {
```

Comment:
If PSR2 is a goal then this short hand should be phase out. A full `if` block is needed to pass.